odd_collector/adapters/druid/mappers/database.py

```diff
@@ -0,0 +1,21 @@
+from typing import List
+
+from odd_models.models import DataEntity, DataEntityGroup, DataEntityType
+
+from odd_collector.adapters.druid.generator import DruidGenerator
+
+
+def to_data_entity_group(
+    generator: DruidGenerator, service_name: str, entities: List[str]
+) -> DataEntity:
+    """
+    :param service_name:
+    :param generator:
+    :param entities - list of Table | View oddrn
+    """
+    return DataEntity(
+        oddrn=generator.get_oddrn_by_path("catalogs"),
+        name=service_name,
```

Comment:
Would it be better to use druid catalog's name for `name` field ? 
For now all DEG from Apache Druid will have name 'druid', i think it can be confusing on UI side

Reply:
Yes make sense. Going to change that 👍

Reply:
Although Druid main catalog name is always `druid` by default. So does it make sense to fill in the host name as the service name?